cloud/deployment/fromfile/fromfile_test.go

```diff
@@ -1802,6 +1828,11 @@ deployment:
   alert_emails:
     - test1@test.com
     - test2@test.com
+  hibernation_schedules:
```

Comment:
not sure if we would need a way to distinguish between no schedules passed to set it to empty vs no hibernation schedules defined to leave it as it is.

The only reason I could think of that we would need it is that users migrating from the earlier CLI version to this one would have an old deployment file, and when they use that with the new CLI version, I think the hibernation schedules will get overwritten to empty for those deployments if already set vs when with older CLI they would remain untouched.

Reply:
we should at least make sure that old files still work. This is kinda of tough because wouldn't you expect the hibernation schedules to go away if they are not present in the file? How would user delete schedules using the file if they didn't?